com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed a UI bug that affected the multi-editing for Importer options.
```

Comment:
Suggestion (if I understand correctly):
`Fixed a UI issue affecting the use of Importer options in multi-editing context.`